contracts/l1/Bridge.sol

```diff
@@ -214,8 +232,9 @@ contract Bridge is IBridge, VersionedInitializable {
         address recipient,
         uint256 amount
     ) external override {
-        require(recipient != address(0), Errors.B_INVALID_ADDRESS);
         require(amount > 0, Errors.B_INSUFFICIENT_AMOUNT);
+        //check if enough rewards are available on the bridge before consuming the message from l2
+        require(getAvailableRewards() >= amount, Errors.B_NOT_ENOUGH_REWARDS);
```

Comment:
Why this is required?

Reply:
to make it fail fast if there aren't enough rewards (both claimed and pending), if there are enough rewards it will move on to the _transferRewards logic otherwise revert.